default.hbs

```diff
@@ -16,8 +16,14 @@
   <footer class="site-footer">
     <a class="subscribe" href="{{@blog.url}}/rss/">Subscribe!</a>
     <div class="inner">
-       <section class="copyright">All content copyright <a href="{{@blog.url}}/">{{@blog.title}}</a> &copy; {{date format="YYYY"}} &bull; All rights reserved.</section>
-       <section class="poweredby">Proudly published with <a href="https://ghost.org">Ghost</a></section>
+       <section class="copyright">
```

Comment:
Is this copyright text more of a paragraph (`p`) than a `section`? `section`s [create new section in the document outline](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#Sectioning_content), as do `footer`s which this is already wrapped in.

Reply:
I think that works. I't fundamentally different than the subscribe. also the Subscribe is going to get more complex once we add support for email.